osquery_rules/linux/osquery_logins_non_office.py

```diff
@@ -0,0 +1,31 @@
+import ipaddress
+
+# Note: The IP Network below is only an example
```

Comment:
Do we have a list of rules that essentially require user configuration? Or are those the rules that we disable by default?

Reply:
We use the following tag to group them:

```
Tags:
  - Configuration Required
```